pkgs/misc/cbeams/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi, blessings, docopt }:
+
+buildPythonPackage rec {
+  pname = "cbeams";
+  version = "1.0.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0y5q2d4kfrlihj00a4sfwjn9g6a0bwlmnv22xssyldyhlkj1p0aj";
+  };
+
+  propagatedBuildInputs = [ blessings docopt ];
+
+  postPatch = ''
+    sed -i "s/docopt==0.6.1/docopt/; s/blessings==1.6/blessings/" setup.py
```

Comment:
Are you sure that unpinning this repositories doesn't break the package? I'm sure there was a reason for upstream to pin them in the first place. If you think there shouldn't be a reason to do so, please contact upstream and perhaps add a comment explaining this.

Reply:
@doronbehar: Thanks for your feedback. I changed the patch to a `>=` version selection. The hard-pinned version are kind of outdated and I successfully tested the program with the current dependencies, which do not have so much greater version numbers. What is your opinion on this?

Reply:
Even with a `>=`, I just want to see that upstream is aware of this. Could you please open an issue on this at https://github.com/tartley/cbeams/issues/new ?

Reply:
@doronbehar: There is currently even a PR to address exactly this issue. https://github.com/tartley/cbeams/pull/1

Reply:
Great! So please add a comment above the `postPatch` with a link to it.

Reply:
@doronbehar: Thanks. Otherwise, we can wait until this is merged upstream. Afterwards I would update the PR.

Reply:
The issue was closed upstream and I have updated the PR. There are no hacky patches left.